test/extended/deployments/util.go

```diff
@@ -625,24 +614,17 @@ func readDCFixture(path string) (*appsapi.DeploymentConfig, error) {
 	if err != nil {
 		return nil, err
 	}
-
-	dcv1 := new(appsapiv1.DeploymentConfig)
-	err = yaml.Unmarshal(data, dcv1)
+	content, err := kyaml.ToJSON(data)
 	if err != nil {
 		return nil, err
 	}
-
-	dc := new(appsapi.DeploymentConfig)
-	err = legacyscheme.Scheme.Convert(dcv1, dc, nil)
-	return dc, err
-}
-
-func readDCFixtureOrDie(path string) *appsapi.DeploymentConfig {
-	data, err := readDCFixture(path)
-	if err != nil {
-		panic(err)
-	}
-	return data
+	appsScheme := runtime.NewScheme()
```

Comment:
@deads2k @tnozicka double check? not sure why the tests were failing

Reply:
> @deads2k @tnozicka double check? not sure why the tests were failing

what error were you getting?

Reply:
```
    Expected error:
        <*runtime.notRegisteredErr | 0xc42194cfa0>: {
            gvk: {
                Group: "",
                Version: "v1",
                Kind: "DeploymentConfig",
            },
            target: nil,
            t: nil,
        }
        no kind "DeploymentConfig" is registered for version "v1"
    not to have occurred
```

Reply:
oh, the fixtures might be oapi...

Reply:
fixed. (converted all test fixtures to grouped api)